README.md

```diff
@@ -233,6 +236,99 @@ crud.insert_object('customers', {
 ...
 ```
 
+### Insert many
+
+```lua
+-- Insert batch of tuples
+local result, err = crud.insert_many(space_name, tuples, opts)
+-- Insert batch of objects
+local result, err = crud.insert_object_many(space_name, objects, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space to insert an object
+* `tuples` / `objects` (`table`) - array of tuples/objects to insert
+* `opts`:
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+  * `fields` (`?table`) - field names for getting only a subset of fields
+  * `stop_on_error` (`?boolean`) - stop on a first error and report error
+    regarding the failed operation and error about what tuples were not
+    performed, default is `false`
+  * `rollback_on_error` (`?boolean`) - any failed operation will lead to
+    rollback on a storage, where the operation is failed, report error
+    about what tuples were rollback, default is `false`
+
+Returns metadata and array contains inserted rows, array of errors.
+Error object can contain field `operation_data` field.
+
+This field can contain:
```

Comment:
The error format seems confusing and overcomplicated. If I've got the error and rolled back, what would I get in return? Only rolled back records? Both error tuple and rolled backed records? As {error_tuple, rolled_tuple_1, rolled_tuple_2, ...} or {error_tuple, {rolled_tuple_1, rolled_tuple_2, ...}}? How can I distinct error tuple from rolled back ones? Same is about stopped by error case.

Reply:
Oh, it seems that if we get an error on stop_on_error and rollback enabled, we'll get three different error with different operational data. We should describe it in README so it should be easy to understand.

But, to be honest, I still don't like an idea of `operation_data` to be tuple or array of tuples.